pkg/pillar/zedUpload/Makefile

```diff
@@ -0,0 +1,75 @@
+# Makefile
```

Comment:
Hey @zed-rishabh -- why is this extra Makefile here? Why can't we hook it up to existing makefile infrastructure either at pillar or EVE level.

Proliferation of Makefiles is kind of an evil thing